AsyncDisplayKit/ASMultiplexImageNode.mm

```diff
@@ -353,9 +355,11 @@ - (id)_nextImageIdentifierToDownload
 {
   OSSpinLockLock(&_imageIdentifiersLock);
 
+  id _currentImageIdentifier = _failedImageIdentifier != nil ? _failedImageIdentifier : _loadedImageIdentifier;
```

Comment:
nit:  the underscore prefix signifies an instance variable, but we're creating a local variable here — let's change this from `_currentImageIdentifier` to `currentImageIdentifier`